src/lib/core/data-table/data-table.ts

```diff
@@ -196,6 +215,32 @@ export class CdkTable<T> implements CollectionViewer {
     }
 
     this._isViewInitialized = true;
+
+    this._StickyRows.changes.subscribe((queryList: QueryList<CdkStickyRow>) => {
+        let topOffset = 0;
+
+        if (queryList.toArray().length) {
+        this.stickyRowsEnabled = true;
+          setTimeout(() => {
+            /** This needs to be called AFTER the "ckd-table-sticky-rows-enabled" class
+             *  gets added to the table body to properly calculate the table's height.
+             */
+            this.tableHeight  = this.elementRef.nativeElement.getBoundingClientRect().height;
+            this._changeDetectorRef.markForCheck();
+          });
+        } else {
+          this.stickyRowsEnabled = false;
+        }
+
+        queryList.forEach((item) => {
```

Comment:
This certainly works for multiple sticky rows, I will make an example in the demo!